fearless/Common/Model/ChainRegistry/ChainModel.swift

```diff
@@ -10,6 +10,17 @@ struct ChainModel: Equatable, Codable, Hashable {
         let overridesCommon: Bool
     }
 
+    struct ExternalApi: Codable, Hashable {
+        let type: String
+        let url: URL
+    }
+
+    struct ExternalApis: Codable, Hashable {
```

Comment:
I would rename it to `ExternalApiSet` if only it wasn't confusing. `ExternalApiKit` maybe?